Pod/Classes/TMPodspec.h

```diff
@@ -18,9 +18,19 @@
 @property (nonatomic, readonly) NSString *version;
 
 /**
- *  @see http://guides.cocoapods.org/syntax/podspec.html#summary
+ *  @see http://guides.cocoapods.org/syntax/podspec.html#authors
  */
-@property (nonatomic, readonly) NSString *summary;
+@property (nonatomic, readonly) NSArray /* <TMAuthor> */ *authors;
```

Comment:
I based the spacing on UIAlertView.h’s which is for a different purpose I guess, so I can change this.

```objc
@property(nonatomic,assign) id /*<UIAlertViewDelegate>*/ delegate;    // weak reference
```